docs/contribute-docs.md

```diff
@@ -30,26 +30,47 @@ To build the MyST guide documentation:
 
 This will build the documentation locally so that you can preview what your changes will look like.
 
-## How this relates to the documentation at mystmd.org
+## How to update the content at mystmd.org
+
+When you change the content here or in most other MyST repositories, it will _not_ automatically update mystmd.org. You must take one extra step, described below.
+
+The content at mystmd.org is automatically updated with [a GitHub Action to deploy to our Vercel project](https://github.com/jupyter-book/mystmd.org/blob/main/.github/workflows/deploy.yml). When that action is run, it will pull the latest docs content from several MyST repositories, and push them to our Vercel deployment.
+
+To trigger that action, you can either:
+
+1. **Trigger a workflow dispatch**. Go to [the Vercel action page](https://github.com/jupyter-book/mystmd.org/actions/workflows/deploy.yml). Click on *Run Workflow* and it will run.
+2. **Push a commit to `main`**. Any new commit to the `main` branch will trigger this action.
+
+In either case, once the action completes the content at `mystmd.org` will be updated.
```

Comment:
A bit weird, because the "will be updated" part is already done. So, would consider removing the sentence, or changing to something like "Once the action completes, the content at `mystmd.org` should be up to date."